repos/system_upgrade/el7toel8/actors/restrictedpcisscanner/tests/unit_test_restricted_pcis_scanner.py

```diff
@@ -0,0 +1,194 @@
+from functools import partial
+
+import mock
+import pytest
+import six
+
+from leapp.libraries.actor import restrictedpcisscanner
+from leapp.libraries.common.testutils import CurrentActorMocked, produce_mocked
+from leapp.libraries.stdlib import api
+from leapp.models import RestrictedPCIDevice, RestrictedPCIDevices
+
+if six.PY2:
+    from mock import mock_open
+else:
+    from unittest.mock import mock_open
+
+unsupported_driver_names_example = {
+    "devices": {
+        "3w-9xxx": {
+            "pci_id": "nan",
+            "driver_name": "3w-9xxx",
+            "device_name": "3w-9xxx",
+            "available_rhel7": 1,
+            "supported_rhel7": 0,
+            "available_rhel8": 0,
+            "supported_rhel8": 0,
+            "available_rhel9": 0,
+            "supported_rhel9": 0,
+            "comment": "nan",
+        },
+        "3w-sas": {
+            "pci_id": "nan",
+            "driver_name": "3w-sas",
+            "device_name": "3w-sas",
+            "available_rhel7": 1,
+            "supported_rhel7": 0,
+            "available_rhel8": 0,
+            "supported_rhel8": 0,
+            "available_rhel9": 0,
+            "supported_rhel9": 0,
+            "comment": "nan",
+        },
+    }
+}
+
+
+unsupported_pci_ids_example = {
+    "devices": {
+        "0x1000:0x0060": {
+            "pci_id": "0x1000:0x0060",
+            "driver_name": "megaraid_sas",
+            "device_name": "SAS1078R",
+            "available_rhel7": 1,
+            "supported_rhel7": 1,
+            "available_rhel8": 0,
+            "supported_rhel8": 0,
+            "available_rhel9": 0,
+            "supported_rhel9": 0,
+            "comment": "nan",
+        },
+        "0x1000:0x0064": {
+            "pci_id": "0x1000:0x0064",
+            "driver_name": "mpt2sas",
+            "device_name": "SAS2116_1",
+            "available_rhel7": 1,
+            "supported_rhel7": 1,
+            "available_rhel8": 0,
+            "supported_rhel8": 0,
+            "available_rhel9": 0,
+            "supported_rhel9": 0,
+            "comment": "nan",
+        },
+    }
+}
+
+
+def blank_fn(*args, should_return, **kwargs):
```

Comment:
Python3 syntax

Reply:
Yes, sad to have this. Travis was very slow, so I didn't captured this before requesting a review. Will fix this diffinetely